src/views/node/AssociationsTable.vue

```diff
@@ -3,72 +3,107 @@
 -->
 
 <template>
-  <div class="container">
-    <!-- status -->
-    <AppStatus v-if="status" :status="status" />
-
-    <!-- results -->
-    <AppTable
-      :cols="cols"
-      :rows="associations"
-      :per-page="perPage"
-      :start="start"
-      :total="count"
-      :search="search"
-      :disabled="!!status"
-      @per-page="(value) => (perPage = value)"
-      @start="(value) => (start = value)"
-      @search="(value) => (search = value)"
-      @download="download"
-    >
-      <!-- "object" (current node) -->
-      <template #subject="{ cell }">
+  <!-- results -->
+  <AppTable
```

Comment:
This is a good idea. I've added the fade with the latest commits.

I wish that all browsers (in this case I'm assuming you're talking about macos/ios chrome/safari) would just show a scroll bar instead of hiding it for aesthetics.